ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerVnextDemoController.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+import SwiftUI
+
+// MARK: DrawerContentController
+
+class DrawerContentController: DemoController {
+
+    public func actionViews() -> [UIView] {
+        let spacer = UIView()
+        spacer.backgroundColor = .orange
+        spacer.layer.borderWidth = 1
+        spacer.heightAnchor.constraint(greaterThanOrEqualToConstant: 20).isActive = true
+
+        var views = [UIView]()
+        views.append(createButton(title: "Dismiss", action: #selector(dismissButtonTapped)))
+        views.append(createButton(title: "Dismiss (no animation)", action: #selector(dismissNotAnimatedButtonTapped)))
```

Comment:
FYI: When you pull the latest changes to your branch, you'll need to change the usage of the button to use the ButtonVnext closure instead of the selector.